### DISCUSSION TITLE
Cant find server.sh

### DISCUSSION THREAD

**Discussion by Kaushal-Kishore-Gupta** [gitpod-io/openvscode-server#334] [Q&A]
I am setting up OpenVScode on digital ocean server and the guide says to run server.sh script. However I cant find any such script in the latest build

**Comment by filiptronicek** [comment]
Hey there @Kaushal-Kishore-Gupta 👋! Sorry for the outdated docs, we hope to update them soon for all of the different cloud providers, the executable script has been renamed and moved, so instead of doing

```bash
./script.sh
```

you can do 

```bash
 ./bin/openvscode-server
```
